lib/utils/fetchSDP.ts

```diff
@@ -0,0 +1,27 @@
+import { WsSdpPipeline, VideoMedia } from 'media-stream-library'
+
+export const fetchSDP = (wsURI?: string, rtspURI?: string) => {
+  const pipeline = new WsSdpPipeline({
+    ws: { uri: wsURI },
+    rtsp: { uri: rtspURI },
+  })
+
+  return pipeline.sdp.finally(() => {
+    pipeline.close()
+  })
+}
+
+export const fetchTransformationMatrix = (wsURI?: string, rtspURI?: string) => {
+  return fetchSDP(wsURI, rtspURI).then((sdp) => {
+    const videoMedia = sdp.media.find((media): media is VideoMedia => {
+      return media.type === 'video'
+    })
+
+    const transform = videoMedia?.transform
```

Comment:
If `basis` is `sensor`, then use `x-sensor-transform` if it exists, otherwise use `transform`.
If `basis` is `metadata`, use `transform`

Reply:
Fixed?